CHANGELOG

```diff
@@ -0,0 +1,36 @@
+# Changelog
+
+## v0.15.0 (pre-release)
+
+- Updates to workflow
+  - All changes to the workflow have been documented in the [readme](https://github.com/includeos/IncludeOS/blob/dev/README.md)
```

Comment:
Link is to README on this branch - could you make it relative?

Reply:
I've made the link relative. And I've made a lot more changes, mainly following the guidelines here: https://keepachangelog.com/en/1.0.0/
- [unreleased] section at the top
- Subsections like added, changed, removed
- made it an `.md` file so github knows to format the file like markdown
- All headers are now linkable.